pkg/rootless/rootless_unsupported.go

```diff
@@ -29,6 +29,12 @@ func GetRootlessGID() int {
 	return -1
 }
 
+// EnableLinger configures the system to not kill the user processes once the session
+// terminates
+func EnableLinger() (string, error) {
```

Comment:
Can probably drop this

Reply:
Oh nevermind, I'm wrong. Ignore.

Reply:
do you mean the `EnableLinger` function?

Reply:
I thought you'd folded it entirely into the XDGRuntime function, so I assume we didn't need the unsupported version anymore - my fault for not looking closer